conventions/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -91,7 +91,7 @@ if (project.findProperty("otel.stable") == "true") {
 
         // this is needed so that we only consider the current artifact, and not dependencies
         ignoreMissingClasses.set(true)
-        packageExcludes.addAll("*.internal", "*.internal.*")
+        packageExcludes.addAll("*.internal", "*.internal.*", "*.shaded", "*.shaded.*")
```

Comment:
should we update https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md? (and/or our repo copy: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/VERSIONING.md):

> A backwards-incompatible change affects the public API of a module. The public API is any public
class or method that is not in a package which includes the word `internal`.

or maybe we should put the shaded classes under `.internal.shaded`?

Reply:
`.internal.shaded` seems to be the easiest